prboom2/CMakeLists.txt

```diff
@@ -22,6 +52,19 @@ TEST_BIG_ENDIAN(WORDS_BIGENDIAN)
 
 include(GNUInstallDirs)
 
+include(FeatureSummary)
+
+if(VCPKG_TOOLCHAIN)
+    set(CMAKE_FIND_PACKAGE_PREFER_CONFIG TRUE)
```

Comment:
What if we enable this option for all build types? It seems MSYS2 also provides configs.